node/network/protocol/src/peer_set.rs

```diff
@@ -36,9 +36,15 @@ impl PeerSet {
 	/// network service.
 	pub fn get_info(self) -> NonDefaultSetConfig {
 		let protocol = self.into_protocol_name();
+		let max_notification_size = match self {
+			PeerSet::Validation = 1024 * 1024,
```

Comment:
I went for 1MiB because pragmatically speaking that's the current limit, but it could be lower.